src/Storage/Migration/v0.00/01-setup-tables.sql

```diff
@@ -68,6 +68,8 @@ TABLESPACE pg_default;
 
 CREATE INDEX IF NOT EXISTS dataelements_instanceinternalId ON storage.dataelements(instanceInternalId);
 CREATE INDEX IF NOT EXISTS dataelements_instanceguid ON storage.dataelements(instanceGuid);
+CREATE INDEX IF NOT EXISTS dataelements_isharddeleted ON storage.dataelements(id)
```

Comment:
Has this been run manually in all environments?

Reply:
Yes, ref comment in the description.